ledger/jetcoordinator/utils.go

```diff
@@ -55,7 +55,16 @@ func selectByEntropy(entropy core.Entropy, values []core.RecordRef, count int) (
 	sort.SliceStable(hashes, func(i, j int) bool { return bytes.Compare(hashes[i].hash, hashes[j].hash) < 0 })
 
 	selected := make([]core.RecordRef, 0, count)
-	for i := 0; i < count; i++ {
+
+	min := func(a, b int) int {
```

Comment:
This is also incorrect. The function should not produce a valid result if the caller expected more candidates than available.